frontend/src/components/About.vue

```diff
@@ -141,34 +262,41 @@ export default {
           img: require('../assets/pic-jens.jpg'),
           orcid: 'https://orcid.org/0000-0002-9955-6003',
           linkedin: 'https://www.linkedin.com/in/jens-nielsen-57a207181/',
-          content: '<a href="https://bii.dk/team/jens-nielsen" target="_blank">CEO at BioInnovation Institute</a> and <a href="https://www.sysbio.se/labs/nielsen/" target="_blank">Prof. of Systems and Synthetic Biology at Chalmers University of Technology</a>',
-        }, {
+          content:
+            '<a href="https://bii.dk/team/jens-nielsen" target="_blank">CEO at BioInnovation Institute</a> and <a href="https://www.sysbio.se/labs/nielsen/" target="_blank">Prof. of Systems and Synthetic Biology at Chalmers University of Technology</a>',
+        },
+        {
           name: 'Dr. Jonathan Robinson',
           img: require('../assets/pic-jon.jpg'),
           orcid: 'https://orcid.org/0000-0001-8567-5960',
           linkedin: 'https://www.linkedin.com/in/jonathanrob',
           github: 'https://github.com/JonathanRob',
-          content: '<a href="https://bii.dk/team/jonathan-robinson" target="_blank">Scientific Data Developer at BioInnovation Institute</a>',
-        }, {
+          content:
+            '<a href="https://bii.dk/team/jonathan-robinson" target="_blank">Scientific Data Developer at BioInnovation Institute</a>',
+        },
+        {
           name: 'Dr. Hao Wang',
           img: require('../assets/pic-hao.jpg'),
           orcid: 'https://orcid.org/0000-0001-7475-0136',
           linkedin: 'https://www.linkedin.com/in/hao-wang-9a66ba30',
           github: 'https://github.com/Hao-Chalmers',
-          content: '<a href="https://github.com/Hao-Chalmers" target="_blank">Researcher at Chalmers University of Technology</a>',
-        }, {
+          content:
+            '<a href="https://github.com/Hao-Chalmers" target="_blank">Researcher at Chalmers University of Technology</a>',
+        },
+        {
           name: 'Dr. Rui Benfeitas',
           img: require('../assets/pic-rui.jpg'),
           orcid: 'https://orcid.org/0000-0001-7972-0083',
           linkedin: 'https://www.linkedin.com/in/ruibenfeitas/',
           github: 'https://github.com/Benfeitas',
-          content: '<a href="https://www.benfeitas.net" target="_blank">Senior Bioinformatician at NBIS, SciLifeLab</a>',
+          content:
+            '<a href="https://www.benfeitas.net" target="_blank">Senior Bioinformatician at NBIS, SciLifeLab</a>',
```

Comment:
This kind of line breaking does not make the code prettier, since the line after breaking is still longer than 80 characters. I would think the original one looks better. Maybe it is not so easy for Prettier to handle such cases.

Reply:
Thanks for spotting all the weird formatting, @nanjiangshu and @mihai-sysbio . I think most cases can be fixed by improving our prettier configuration (eg the whitespace sensitivity). It will be updated!